npm/templates/deploy.py

```diff
@@ -42,10 +42,11 @@
 }
 npm_registry = npm_repositories[repo_type]
 
-npm_username, npm_password, npm_email = (
+npm_username, npm_password, npm_email, npm_token = (
     os.getenv('DEPLOY_NPM_USERNAME'),
```

Comment:
Do we need to take in `DEPLOY_NPM_USERNAME` and `DEPLOY_NPM_PASSWORD` still?

Reply:
That's a fair point, token authentication doesn't require username/password, I'll delete them